### PR TITLE
Docs/Review #647

### PR DESCRIPTION
**Pull request by Himura2la**
The #647 PR was merged without being checked. This PR is intended to fix this.

Files to check:

- [x] docs\API References.md
- [x] packages\create-resolve-app\README.md
- [x] packages\resolve-scripts\README.md
- [x] examples\hello-world\resolve.config.json
- [x] examples\todo\resolve.config.json
- [x] examples\todo-two-levels\resolve.config.json
- [x] examples\top-list\resolve.config.json
- [x] examples\with-authentication\resolve.config.json
- [x] examples\with-postcss-modules\resolve.config.json
- [x] examples\with-styled-components\resolve.config.json
- [x] packages\resolve-scripts\configs\cli.list.json
- [x] packages\resolve-scripts\configs\env.list.json
- [x] packages\resolve-scripts\configs\message.json
- [x] delete `to_check.txt`

TODO:

- [x] Corrector's Check

### CODE REVIEW

**Comment by Himura2la** [comment]
@MrCheater 

https://github.com/MrCheater/resolve/blob/f89801bbf9b1bdf123c5a0a4842e0cdbb51c3854/packages/resolve-scripts/README.md

> App's configuration can be defined in the resolve.config.json file or through the --config <path/to/json> option.

Which command accepts the `--config` argument?

File: packages/create-resolve-app/README.md [hunk 1]
```diff
@@ -1,86 +1,99 @@
...
+This package creates an empty application based on the reSolve framework ([`hello-world` example](https://github.com/reimagined/resolve/tree/master/examples/hello-world)). Use the `--example` option to create another sample application. reSolve applications are built on the CQRS and Event Sourcing principles and use [React](https://github.com/facebook/react) + [Redux](https://github.com/reactjs/redux) on the client side.
 
-Create reSolve App allows you to specify application blocks (aggregates, read models, and UI part React components present) in a semi-declarative manner. With the `resolve-scripts` package, you do not need to write an API backend manually. Instead, `resolve-scripts` deploys backend and domain services to interact with the client which is wrapped in the `resolve-redux` package for an automated interaction.
+**Create reSolve App** allows you to specify application blocks (aggregates, read models, and UI elements (React components)) in a semi-declarative manner. The `resolve-scripts` package deploys API backend and domain services to interact with the client. The `resolve-redux` package automates the client-server interaction.
```
Comment (Himura2la) [inline packages/create-resolve-app/README.md]: I'm not sure about the `resolve-redux` package, please check this info

Comment (MrCheater) [reply packages/create-resolve-app/README.md]: > Which command accepts the --config argument?

`npm run dev --config=my-custom-resolve.config.json`
`npm run build --config=my-custom-resolve.config.json`

Comment (MrCheater) [reply packages/create-resolve-app/README.md]: > Which command accepts the --config argument?

npm run dev --config=my-custom-resolve.config.json
npm run build --config=my-custom-resolve.config.json

Comment (IhostVlad) [reply packages/create-resolve-app/README.md]: Text okay

**Comment by Himura2la** [comment]
on Paul's check https://hackmd.io/FBJc7BbpSbestUndRjz3Qw

**Comment by Himura2la** [comment]
Ready to be Merged

**Pull request merged**